packages/react-app/src/components/BuilderBadgeCard.jsx

```diff
@@ -1,58 +1,58 @@
 import React, { useState, useEffect } from "react";
-import { Flex, Spinner, Skeleton, SkeletonText, Image } from "@chakra-ui/react";
-import axios from "axios";
+import { Spinner, Image, Flex, Skeleton, SkeletonText } from "@chakra-ui/react";
 import useCustomColorModes from "../hooks/useCustomColorModes";
+import axios from "axios";
 
 const BuilderBadgeCardSkeleton = ({ isLoaded, children }) => (
   <Skeleton isLoaded={isLoaded}>{isLoaded ? children() : <SkeletonText mt="4" noOfLines={4} spacing="4" />}</Skeleton>
 );
 
+// TODO get the actual join date. Should be easy getting the user.create event
 const BuilderBadgeCard = ({ builder, readContracts, builderAddress }) => {
   const [balance, setBalance] = useState({
     loading: true,
     items: [],
   });
 
-  const { borderColor } = useCustomColorModes();
+  const { borderColor, secondaryFontColor } = useCustomColorModes();
 
-  const getImageAddress = async badge => {
+  const getTokenMetaData = async badges => {
+    //const parsed = parseInt(badges, 10);
     const metadata = await axios.get(
-      `https://forgottenbots.mypinata.cloud/ipfs/QmZesNT9tbpaNoy727fYRa7cB936dznKqFtZwNwUSbxJBg/000000000000000000000000000000000000000000000000000000000000000${badge}.json`,
+      `https://forgottenbots.mypinata.cloud/ipfs/QmZesNT9tbpaNoy727fYRa7cB936dznKqFtZwNwUSbxJBg/000000000000000000000000000000000000000000000000000000000000000${badges}.json`,
     );
+    //tokenDatas.push(metadata.data);
     return metadata.data;
   };
 
-  useEffect(() => {
-    const loadBadges = async () => {
-      if (!builderAddress || !readContracts) return;
-      setBalance({
-        loading: true,
-        items: [],
-      });
-
-      const balances = [];
+  const loadBadges = async () => {
+    if (!builderAddress || !readContracts) return;
+    setBalance({
+      loading: true,
+      items: [],
```

Comment:
I moved inside the useEffect to avoid the missing dependency on the useEffect on the 55.